guides/ReleaseNotes.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## New Endpoints: [Wix Blog API](wix-blog/)
```

Comment:
Did you mean to link to the outer Wix Blog item or to the [Blog](wix-blog/blog) artifact?

Reply:
@imnotashrimp this one:

https://dev.wix.com/api/rest/wix-blog